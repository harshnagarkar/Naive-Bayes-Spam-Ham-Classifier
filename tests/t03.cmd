  training_ham.txt training_spam.txt testing_ham3.txt testing_spam.txt 1 
