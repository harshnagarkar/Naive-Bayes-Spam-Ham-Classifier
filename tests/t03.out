ham -232.678 -258.332
ham -170.114 -179.445
ham -39.1118 -43.2052
ham -226.528 -245.175
ham -297.681 -318.745
ham -339.896 -353.933
ham -243.617 -260.238
ham -269.613 -284.939
ham -316.5 -346.563
ham -224.067 -247.578
ham -169.202 -190.268
ham -245.01 -256.677
ham -483.048 -519.377
ham -216.494 -236.32
ham -52.263 -56.2762
ham -401.641 -440.899
ham -449.937 -488.16
ham -0.241914 -1.53769
ham -0.241914 -1.53769
ham -585.429 -591.384
ham -1143.17 -1154.81
ham -739.065 -762.712
ham -504.712 -517.13
spam -149.792 -146.506
spam -108.636 -107.676
ham -221.689 -225.496
ham -391.922 -401.107
spam -94.1671 -92.1079
ham -21.8596 -23.6131
ham -805.333 -821.779
ham -590.725 -609.549
ham -643.487 -660.724
ham -513.057 -525.474
ham -655.955 -674.365
ham -0.241914 -1.53769
1 0.59375 0.628571
